backend/app/api/routes/doc_transformation_job.py

```
@@ -0,0 +1,58 @@
+            invalid_ids.append(jid)
+
+    if invalid_ids:
+        raise HTTPException(
+            status_code=422,
+            detail=f"Invalid UUID(s) provided: {', '.join(invalid_ids)}",
+        )
+
+    crud = DocTransformationJobCrud(session, project_id=current_user.project_id)
+    jobs = crud.read_each(set(job_id_list))
+    jobs_not_found = set(job_id_list) - {job.id for job in jobs}
+    return APIResponse.success_response(DocTransformationJobs(jobs=jobs, jobs_not_found=jobs_not_found))
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)